sklearn/svm/classes.py

```diff
@@ -177,6 +177,11 @@ def fit(self, X, y):
         y : array-like, shape = [n_samples]
             Target vector relative to X
 
+        sample_weight : array-like, shape = [n_samples], optional
+                        Array of weights that are assigned to individual
```

Comment:
Hi, I am afraid that you got this alignment wrong: you can have a look at the lines right above those that you modified: you documentation line should be aligned the same way as the documentation line on the "y" parameter.